### DISCUSSION TITLE
`eslint-plugin-header` rule

### DISCUSSION THREAD

**Discussion by raspy8766** [biomejs/biome#206] [Rule suggestion]
So far outside of a few quirks here and there, we have really enjoyed using Rome and Biomejs! The only eslint configuration we miss, is the ability for new files (or on edit) to auto-add a copyright header (e.g. [eslint-plugin-header](https://www.npmjs.com/package/eslint-plugin-header)). As of now we have a GitHub action we can leverage to audit the repo for file copyright headers, but that adds some extra churn to the CI process.

Is there any chance something like this will be supported natively in biomejs?

**Comment by ematipico** [comment]
Thank you @raspy8766, this seems to be a rule that is required only in particular cases.

I don't think it's a rule we can give high priority to. If you're willing to implement it, we can definitely help you.

**Comment by Netail** [comment]
It's good to mention that bundlers also have plugins to add such banners, e.g. webpack's [banner-plugin](https://webpack.js.org/plugins/banner-plugin/) or [vite-plugin-banner](https://www.npmjs.com/package/vite-plugin-banner)
